pca_cca.go

```diff
@@ -133,21 +133,18 @@ func (c CC) Right(spheredSpace bool) (vecs *mat64.Dense) {
 // correlation analysis. The canonical correlation matrix is calculated using
 // the singular value decompositions of the centered input.
 //
-// CanonicalCorrelations will panic if the inputs x and y do not have the same
-// number of rows. Columns of x and y should be interpretable as two different
-// sets of measurements on the same observations.
-//
-// The slice weights is used to weight the observations. If weights is nil, each
-// weight is considered to have a value of one, otherwise the length of weights
-// must match the number of observations or CanonicalCorrelations will panic.
+// Canonical correlation analysis can be thought of an approach to finding
+// associations between two sets of variables on the same observations. First,
+// correlations between variables within each set are removed by sphereing both
```

Comment:
s/removed by sphereing both datasets/remove in a process called sphering/ (spelling?)